recipes/shelephant/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "shelephant" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "a692acb0373c950e8134ccc356a392f284e5ed3b240ac6fa6b467ed8475b8fb8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - shelephant_dump = shelephant.cli.shelephant_dump:main
+    - shelephant_get = shelephant.cli.shelephant_get:main
+    - shelephant_hash = shelephant.cli.shelephant_hash:main
+    - shelephant_remote = shelephant.cli.shelephant_remote:main
+    - shelephant_rm = shelephant.cli.shelephant_rm:main
+    - shelephant_send = shelephant.cli.shelephant_send:main
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - docopt
```

Comment:
Use the version restrictions in the upstream `setup.py`:

https://github.com/tdegeus/shelephant/blob/main/setup.py#L21